cli/module_registry.go

```diff
@@ -64,11 +64,11 @@ func CreateModuleCommand(c *cli.Context) error {
 		return err
 	}
 	if org == nil {
-		return errors.Errorf("Unable to determine org from orgID(%q) and namespace(%q)", orgIDArg, publicNamespaceArg)
+		return errors.Errorf("unable to determine org from orgID(%q) and namespace(%q)", orgIDArg, publicNamespaceArg)
```

Comment:
```suggestion
		return errors.Errorf("unable to determine org from org-id (%q) and namespace (%q)", orgIDArg, publicNamespaceArg)
```

`org-id` to match usage presented to end users, and spaces in case I am reading string interpolation correctly. If I'm wrong about the spacing, just ignore me!